lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -67,6 +67,13 @@
       - The database extension state
     default: present
     choices: [ "present", "absent" ]
+  cascade:
+    description:
+      - Automatically install/remove any extensions that this extension depends on
+        that are not already installed/removed.
+    type: bool
+    default: no
+    version_added: '2.8'
```

Comment:
Add a couple of examples for "preset" and "absent" state with cascade: true to EXAMPLE block